### DISCUSSION TITLE
opensource local alternative for roboflow.com

### DISCUSSION THREAD

**Discussion by damavand1** [ultralytics/ultralytics#8681] [General]
Hi 

I want to create and manage and annotate items in pictures in my local host laptop without upload dataset 

Is there any opensource software for this ?

tnx

**Comment by glenn-jocher** [comment]
Hello!

For local dataset management and annotation, you might want to explore **CVAT** (Computer Vision Annotation Tool) by Intel. It's an open-source web-based tool that allows you to annotate images and videos for computer vision tasks. You can run it on your local machine without the need to upload your dataset to a third party. Here's how you can get started with Docker:

```bash
docker-compose up -d
```

For more details and installation instructions, check out their GitHub repository: [CVAT GitHub](https://github.com/openvinotoolkit/cvat).

Hope this helps! 😊
